rust/cubesql/cubesql/src/mysql/mod.rs

```diff
@@ -70,71 +69,62 @@ impl Backend {
             _ => false,
         };
 
-        if query_lower.eq("set autocommit=1, sql_mode = concat(@@sql_mode,',strict_trans_tables')")
+        let stmt = parse_sql_to_statement(&query)?;
```

Comment:
This will not work, because the parser still doesn't support some cases.

show full tables from db
show columns